### PR TITLE
Add a roadmap page, with the basic roadmap for SymPy outlined

### PR DESCRIPTION
**Pull request by asmeurer**
CC @scopatz @certik 

Are there any things that are missing? Please critique my text. 

Unfortunately, because of the way the webpage is generated, the diff is quite huge. Please look at the files in the `templates` directory, particularly [`templates/roadmap.html`](https://github.com/sympy/sympy.github.com/pull/95/files#diff-dde13ba3d4360a814cafa7cbc9e6183c).

### CODE REVIEW

File: cs/roadmap.html [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+      <section class="row" id="main">
+        <div class="col span_7 clr small_span_4">
+          
+          <article id="content">
+            
+
+<h1 class="title">Roadmap</h2>
+<div class="inner">
+  <p class="first">
+    Broadly speaking, any form of poorly symbolic mathematical computation is
```
Comment (scopatz) [inline cs/roadmap.html]: I don't understand what "poorly" means in this context

File: cs/roadmap.html [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+
+
+<h3 class="title">SymPy Live and SymPy Gamma</h3>
+<div class="inner">
+  <p class="first">
+    <a href="http://live.sympy.org/">SymPy Live</a> is a web
+      application that runs a full Python session with SymPy in the browser.
+      It is also used in
+      the <a href="http://docs.sympy.org/latest/index.html">SymPy
+      documentation</a> to allow readers to interactively play with the
```
Comment (scopatz) [inline cs/roadmap.html]: `play -> experiment`

Comment (scopatz) [inline cs/roadmap.html]: or `play -> evaluate`

**Comment by scopatz** [comment]
Thanks @asmeurer! Looks reasonable, a few inline comments

File: cs/roadmap.html (hunk 1 above)
Comment (asmeurer) [inline cs/roadmap.html]: It means I misspelt "purely" and wasn't paying attention when my spellchecker fixed it.

Comment (scopatz) [inline cs/roadmap.html]: Ahh makes sense :)

**Comment by asmeurer** [comment]
@scopatz is this the sort of thing we're looking for, or do we need something else as well?

**Comment by scopatz** [comment]
I think that this is "good enough." We just needed something that listed the project goals and directions.  I think that this satisfies that amply.  

My only comment is that this list should be as broad as possible. So anything that is vaguely on the list of roadmap interest areas should be included. This contains all of the ones I know of, but maybe there are others?

I am for merging this as it is.

**Comment by asmeurer** [comment]
I was hoping the last item was sort of a graball, but maybe I should add a "general improvements" idea at the end (there definitely are things that could be done that aren't algorithms, like bugfixes and documentation and so on).

**Comment by scopatz** [comment]
Yeah, I think General Software Development Improvements would be welcome too and is distinct from algorithmic improvements

**Comment by asmeurer** [comment]
And also the first paragraph.

**Comment by asmeurer** [comment]
OK take a look at the most recent commit.

File: cs/roadmap.html [hunk 3]
```diff
@@ -145,6 +145,17 @@ <h3 class="title">General Algorithmic Improvements</h3>
 </div>
 
 
+<h3 class="title">General Improvements</h3>
```
Comment (scopatz) [inline cs/roadmap.html]: I would call these all "Software Development Improvements." The word "General" is too general.

File: cs/roadmap.html [hunk 4]
```diff
@@ -145,6 +145,17 @@ <h3 class="title">General Algorithmic Improvements</h3>
...
+<div class="inner">
+  <p class="first">
+    Aside from the above specific items, we are always aiming to
+    improve SymPy in many ways. This includes things like fixing bugs, better
+    documentation, improved development tooling, new features, and better
+    community outreach.
```
Comment (scopatz) [inline cs/roadmap.html]: Maybe make this a unordered list?

Comment (asmeurer) [inline cs/roadmap.html]: How do you make something in prose an unordered list?

Comment (scopatz) [inline cs/roadmap.html]: `<ul><li>item 0</li><li> item 2</li></ul>`

Comment (asmeurer) [inline cs/roadmap.html]: Oh is that what ul stands for?

Comment (scopatz) [inline cs/roadmap.html]: yup! And `<ol>` is for ordered list :)

**Comment by scopatz** [comment]
LGTM!

**Comment by asmeurer** [comment]
Going to leave this open for just a little bit in case anyone else wants to have a look. Will merge at the end of the week.

**Comment by scopatz** [comment]
Great!

**Pull request merged**